Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -766,8 +766,16 @@ bool CStaticFunctionDefinitions::IsElementInWater(CClientEntity& Entity, bool& b
         case CCLIENTPLAYER:
         {
             CClientPed& Ped = static_cast<CClientPed&>(Entity);
-            bInWater = Ped.IsInWater();
-            break;
+            if (Ped.GetOccupiedVehicle())
+            {
+                bInWater = Ped.GetOccupiedVehicle()->IsInWater();
+                break;
+            }
+            else
+            {
+                bInWater = Ped.IsInWater();
+                break;
+            }
```

Comment:
Have you tried to put the 'break' down here? What would you put your case there?

Reply:
This is the pattern that this repository uses. I don't think that changing it would be a good choice. Clone this repo. to see how things work - it's an important decision to make when it comes to reviewing a PR.

Reply:
not sure if this was the point being made - but it would be cleaner with a single break, infact the whole condition is perfectly readable as two lines:

```cpp
bInWater = Ped.GetOccupiedVehicle() ? Ped.GetOccupiedVehicle()->IsInWater() : Ped.IsInWater();
break;
```

down to preference I guess